serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1632,14 +1632,15 @@ export class _Exporter {
             if (node.name !== "__root__") {
                 return false;
             }
+
             // Transform
             let matrix = node.getWorldMatrix();
             let matrixToLeftHanded = Matrix.Compose(this._convertToRightHandedSystem ? new Vector3(-1, 1, 1) : Vector3.One(), Quaternion.Identity(), Vector3.Zero());
             let matrixProduct = matrix.multiply(matrixToLeftHanded);
             let matrixIdentity = Matrix.IdentityReadOnly;
 
             for (let i = 0; i < 16; i++) {
-                if (Math.abs(matrixProduct.m[i] - matrixIdentity.m[i]) > Epsilon) {
+                if (matrixProduct.m[i] * matrixIdentity.m[i] < 0) { // checking if they have the same sign

```

Comment:
I wonder if we can just check the matrix determinant to figure this out instead of all this math?

Reply:
I tried but it is not working :(

Reply:
really? that's surprising

Reply:
actually it worked, I'm simply dumb